tests/integration/components/frost-object-browser-test.js

```diff
@@ -230,4 +230,30 @@ describe(test.label, function () {
       expect(displayFilter).to.have.callCount(1)
     })
   })
+
+  describe('when user mouse enters/exists the facets area', function () {
+    const myHook = 'myHook'
+    const filterHook = 'filterHook'
+
+    beforeEach(function () {
+      this.set('myHook', myHook)
+      this.set('filterHook', filterHook)
+
+      this.render(hbs`
+        {{frost-object-browser
+          hook=myHook
+          content=(component 'mock-content' class='mock-content')
+          controls=(component 'mock-controls' class='mock-controls')
+          filters=(component 'mock-filters' class='mock-filters' hook=filterHook)
+        }}
+      `)
+      $hook(filterHook).css('padding-left', '10px')
+      $hook(filterHook).css('padding-bottom', '1500px')
+    })
```

Comment:
Should any series of these calls be using `wait()` and `andThen()` calls?  Line 250 and 251 rely on the completion of line 242 and 254 relies on all of them.  But there isn't anything orchestrating this specific order.